app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,21 @@
+class RelationshipsController < ApplicationController
+  
+  before_filter :require_logged_in
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def destroy
+    relationship = Relationship.find(params[:id])
+    relationship.delete if relationship.follower == current_user
+    redirect_to people_path
+  end
+
+  def create
+    leader = User.find(params[:leader_id])
+    Relationship.create(leader_id: params[:leader_id], follower: current_user) if current_user.can_follow?(leader)
```

Comment:
Since you have the `leader` object, you could also do:

```ruby
Relationship.create(leader: leader, follower: current_user) if current_user.can_follow?(leader)
```